spec/routing/page_not_found_error_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe 'Page Not Found', type: :routing do
+  context 'page exists' do
+    it 'should not show page not found error page' do
+      expect(get: '/en').to be_routable
+    end
+  end
+
+  context 'page does not exist' do
+    it 'should not be routable' do
+      expect(get: '/en/non-existent').to_not be_routable
+    end
```

Comment:
Remove the 'should' from the specs. This is suggestion is based on the better specs site http://betterspecs.org/#should